app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    split_command = command.split()
+    if split_command[1] != split_command[2]:
+        with open(split_command[1], "r") as file_in, \
+                open(split_command[2], "w") as file_out:
```

Comment:
what is split_command[1], split_command[2]?
create separated variables for it and call it on demand